client/shipping-label/views/purchase/steps/packages/add-item.js

```diff
@@ -27,34 +25,20 @@ const AddItemDialog = ( {
 	};
 
 	const renderRadioButton = ( pckgId, itemIdx, item ) => {
```

Comment:
Seems like we can rename this now that it doesn't render radio buttons. Perhaps `renderMovePackageControl` or similar?